### PR TITLE
Allow older version of ssl and older ciphers

### PR DESCRIPTION
**Pull request by bdraco**
related issue https://github.com/home-assistant/core/issues/91130

### CODE REVIEW

**Review by hunterjm — APPROVED** [review]

**Comment by bdraco** [comment]
All the cameras I ordered should show up this week. 

I may have other PRs coming so I'll leave this as a draft until everything shows up and I work through each camera in case there is more

**Comment by bdraco** [comment]
The other cameras I bought on ebay that showed up turned out be the wrong ones (wasn't even IP) or required windows to configure (which I don't have but can borrow a laptop after the 21st).  I ordered a few different ones but in the mean time I didn't want to hold up https://github.com/home-assistant/core/issues/91130 so probably good to do a release with this one and I'll do another round once the other cameras I ordered in https://github.com/home-assistant/core/issues/90912#issuecomment-1506086622 show

**Comment by hunterjm** [comment]
Ok, I'll merge this and ping you when I cut a release.

**Pull request merged**
